pkg/configutil/once.go

```diff
@@ -24,6 +25,10 @@ func LoadConfigOnce() (*config.Config, error) {
 	var err error
 	configOnce.Do(func() {
 		configInfo, err = config.LoadConfig()
+		// set default value
+		if configInfo.EnvelopeType == "" {
+			configInfo.EnvelopeType = envelope.JWS
+		}
```

Comment:
```suggestion
        evelopeType := strings.ToLower(configInfo.EnvelopeType)
		if evelopeType != "jws" && evelopeType != "cose" {
			configInfo.EnvelopeType = envelope.JWS
		}
```
Same here. If a user modifies configInfo.EnvelopeType in the config.json to some random string or just "   ", configInfo.EnvelopeType = envelope.JWS will be bypassed.

Reply:
Thanks for your suggestion! I already considered this issue. For `LoadConfig` function we don't need to handle the error. We should handle it when we really use it.